index.html

```diff
@@ -41,5 +41,19 @@
     </a>
 </div>
 </div>
-    </body>
+
+<div class="ui divider"></div>
+<div class="ui grid">
```

Comment:
We were just looking at this, can you wrap this grid in a container `<div class="ui container">`. This will keep the body of content together and add padding to the sides.